packages/client/src/BulkApi.ts

```diff
@@ -0,0 +1,87 @@
+import { HttpClientErrors, PartitionsResponse } from "@lionweb/repository-common";
+import { LionWebJsonChunk } from "@lionweb/validation";
+import { ClientResponse, RepositoryClient } from "./RepositoryClient.js";
+import { CreatePartitionsResponse, DeletePartitionsResponse, IdsResponse, RetrieveResponse, StoreResponse } from "./responses.js";
+
+export class BulkApi {
+    client: RepositoryClient
+    
+    constructor(client: RepositoryClient) {
+        this.client = client
+    }
+
+    async listPartitions(): Promise<PartitionsResponse> {
+        console.log(`BulkApi.listPartitions`)
+        const partitionsResponse: PartitionsResponse = await this.client.getWithTimeout<PartitionsResponse>("bulk/listPartitions", { body: {}, params: "" })
+        return partitionsResponse
+    }
+
+    async createPartitions(data: LionWebJsonChunk): Promise<ClientResponse<CreatePartitionsResponse>> {
+        console.log(`BulkApi.createPartitions`)
```

Comment:
I would remove the logging or add a flag to control it

Reply:
Done